docs/collections.md

```diff
@@ -0,0 +1,138 @@
+  </tr>
+  <tr>
+    <td><code>label</code></td>
+    <td>What the post type will be called in the admin UI.</td>
+  </tr>
+  <tr>
+    <td><code>folder</code></td>
+    <td>Where files of this type are stored, relative to the repo root.</td>
+  </tr>
+  <tr>
+  <td><code><em>files</em></code></td>
+    <td>Where files of this type are stored, relative to the repo root. If you want to edit specific files, see the "Data Files" section below.</td>
+  </tr>
+  <tr>
+    <td><code>create</code></td>
+    <td>Set to <code>true</code> to allow users to create new files in this collection.
+</td>
+  </tr>
+  <tr>
+    <td><code>slug</code></td>
+    <td>Template for filenames. <code>{{year}}</code>, <code>{{month}}</code>, and <code>{{day}}</code> will pull from the post's <code>date</code> field or save date. <code>{{slug}}</code> is a url-safe version of the post's <code>title</code>. Default is simply <code>{{slug}}</code>.
+</td>
+  </tr>
+  <tr>
+    <td><code>fields</code></td>
+    <td>Fields listed here are shown as fields in the content editor, then saved as front matter at the beginning of the document (except for <code>body</code>, which follows the front matter). Each field contains the following properties:
+      <ul>
+        <li><code>label</code>: Field label in the editor UI.</li>
+        <li><code>name</code>: Field name in the document front matter.</li>
+        <li><code>widget</code>: Determines UI style and value data type (details below).</li>
+        <li><code>default</code> (optional): Sets a default value for the field.</li>
+      </ul>
+    </td>
+  </tr>
+</table>
+
+As described above, the `widget` property specifies a built-in or custom UI widget for a given field. The first field in the example, `layout`, uses a `hidden` widget. This widget will not show in the editor UI, but will be saved with the `default` value (assuming it's been set) in the document front matter. The rest of the widgets work as follows:
+
+Widget | UI | Data Type
+--- | --- | ---
+`string` | text input | string
+`datetime` | date picker widget | ISO date string
+`image` | file picker widget with drag-and-drop | file path saved as string, image uploaded to media folder
+`number` | text input with `+` and `-` buttons | number
+`markdown` | rich text editor with raw option | markdown-formatted string
+
+Based on this example, you can go through the post types in your site and add the appropriate settings to your `config.yml` file. Each post type should be listed as a separate node under the `collections` field.
+
+## Filter
+The entries for any collection can be filtered based on the value of a single field. The example collection, below, would only show post entries with the value "en" in the language field.
+
+``` yaml
+collections:
+  - name: "posts"
+    label: "Post"
+    folder: "_posts"
+    filter:
+      field: language
+      value: en
+    fields:
+      - {label: "Language", name: "language"}
+```
+
+## Files
```

Comment:
Change to "Files Type Collections", and move below the "Folder Type Collections section", just before the table of properties.